during_the_workshop.md

```diff
@@ -4,9 +4,9 @@
 ### Step 1 - Read your brief and user research report
 Choose one of the following three scenarios:
 
-1. [Brief & user research report for Inner-city Travel Advisor](./scenarios/inner-city_travel_advisor.md)
-2. [Brief & user research report for Project Management Tool](./scenarios/project_management.md)
-3. [Brief & user research report for Food Delivery App](./scenarios/food_delivery_app.md)
+1. [Brief & user research report for Holiday Booking Website](./scenarios/holiday_booking_portal.md)
+2. [Brief & user research report for Food Delivery App](./scenarios/food_delivery_app.md)
+3. [Brief & user research report for Inner-city Travel Advisor](./scenarios/inner-city_travel_advisor.md)
```

Comment:
Unnecessary to reorder, but I'm genuinely curious for a non-scientific experiment as to whether a different ordering changes the breakdown of choices at all